api.raml

```diff
@@ -527,15 +527,30 @@ securitySchemes:
               application/json:
                 type: lib.DifferentialAbundanceStatsResponse
 
+  /correlationassayassay/visualizations:
+    displayName: Visualizations for discovering correlations between various assay data
+
+    /bipartitenetwork:
+      post:
+        description: Returns data required to create a bipartite network from a correlation matrix.
+        body:
+          application/json:
+            type: lib.CorrelationAssayAssayBipartitenetworkPostRequest
+        responses:
+          200:
+            body:
+              application/json:
+                type: lib.BipartiteNetworkPostResponse
+
   /correlationassaymetadata/visualizations:
     displayName: Visualizations for discovering corrleations between assay data and sample metadata
 
     /bipartitenetwork:
       post:
-        description: Returns data required to create a volcanoplot from a differential abundance analysis.
+        description: Returns data required to create a bipartite network from a correlation matrix.
         body:
           application/json:
-            type: lib.CorrelationBipartitenetworkPostRequest
+            type: lib.CorrelationAssayMetadataBipartitenetworkPostRequest
```

Comment:
this change is the only that has me nervous. do we think this will break anything front end if i merge?

Reply:
i don't think so. If we merge and tag then we can see on my dev site if anything broke. I can then fix any frontend issues tomorrow